openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -17,7 +17,7 @@
     "preinstall": "cd ../../../../.. && yarn install --frozen-lockfile",
     "postinstall": "yarn run build-check",
     "pre-commit": "lint-staged --concurrent false",
-    "test": "jest --passWithNoTests --maxWorkers=3 --silent",
+    "test": "jest --passWithNoTests --maxWorkers=3",
```

Comment:
Do not remove this as we don't want the test logs in production environment.

Reply:
reverted, accidentally pushed